### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by mohituniyal**

### CODE REVIEW

File: docs/source/About OpenReview.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Read more about OpenReview
```
Comment (mspector) [inline docs/source/About OpenReview.rst]: can the name of this file be changed to something without spaces?

**Comment by melisabok** [comment]
Add a link to the documentation in the README file.

File: docs/source/About OpenReview.rst (hunk 1 above)
Comment (mohituniyal) [reply docs/source/About OpenReview.rst]: Done. Changed name of "About OpenReview.rst" to "about.rst" and "autodoc.rst" to "api.rst".

**Comment by melisabok** [comment]
It looks good to me, @mspector  do you want to do a final check?

File: openreview/tools.py [hunk 2]
```diff
@@ -91,7 +90,6 @@ def build_groups(conference_group_id, default_params=None):
 
     return sorted(groups.values(), key=lambda x: len(x.id))
 
-@deprecated
```
Comment (mspector) [inline openreview/tools.py]: are we giving up on this for now?

File: openreview/tools.py [hunk 3]
```diff
@@ -626,17 +615,12 @@ def recruit_reviewer(client, email, first,
 
     hash_seed: a random number for seeding the hash.
     recruit_message: a formattable string containing the following string variables:
-        name
-        accept_url
-        decline_url
+    name
```
Comment (mspector) [inline openreview/tools.py]: I think these tabs should stay, or there should be some other way of differentiating `name`, `accept_url`, and `decline_url` from the function variables (these three are variables to the formattable string, not the function)

File: openreview/tools.py (hunk 2 above)
Comment (melisabok) [reply openreview/tools.py]: Yes, it means to add a new library to the setup.py, I don't know if it is worth. When we have more deprecated functions we could add it.

**Pull request merged**